### PR TITLE
feat: add service build info metrics

### PR DESCRIPTION
**Pull request by katcipis**

### CODE REVIEW

File: service/metrics.go [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+func SampleBuildInfo() {
+	goVersion := "undefined"
+	revision := "undefined"
+
+	goBuildInfo, ok := debug.ReadBuildInfo()
+	if ok {
+		goVersion = goBuildInfo.GoVersion
+		for _, buildSetting := range goBuildInfo.Settings {
+			if buildSetting.Key == "vcs.revision" {
+				revision = buildSetting.Value
```
Comment (guilherme-santos) [inline service/metrics.go:25]: This is the go revision, right? Shouldn't we also add our service version?

Comment (katcipis) [reply service/metrics.go:25]: not it is not, this is our service version (from vcs/git). The go version is goBuildInfo.GoVersion.

Comment (katcipis) [reply service/metrics.go:25]: Reference:https://pkg.go.dev/runtime/debug#BuildSetting

"vcs.revision: the revision identifier for the current commit or checkout"

Comment (guilherme-santos) [reply service/metrics.go:25]: Mmh, shit! I always injected in the makefile, I didn't know about that.

**Review by guilherme-santos — APPROVED** [review]

File: service/metrics.go (hunk 1 above)
Comment (katcipis) [reply service/metrics.go:25]: All good :-), this is new stuff that came on Go 1.18 (which now it is not that new actually...getting old sux :rofl: )

**Pull request merged**